packages/grafana-ui/src/themes/GlobalStyles/hacks.ts

```diff
@@ -0,0 +1,21 @@
+import { css } from '@emotion/react';
+
+export interface Hacks {
+  hackNoBackdropBlur?: boolean;
+}
+
+export function getHacksStyles(hacks: Hacks) {
+  return css([
+    /**
+     * Disables all backdrop blur effects to improve performance on extremely
+     * resource constrained devices.
+     *
+     * Controlled via the `noBackdropBlur` feature toggle in Grafana
+     */
+    hacks.hackNoBackdropBlur && {
+      '*, *:before, *:after': {
+        backdropFilter: 'none !important',
+      },
+    },
+  ]);
```

Comment:
Not sure how we handle these nowdays, but maybe comment this as https://github.com/internal?

Not sure about the naming here, would `Overrides` work?

Reply:
These styles functions aren't exported from the package so I don't really see the need for `@internal`. At least, none of the other getFooStyles functions have it.

I'm not precious about the naming here, but I thought it useful to acknowledge the smell of this.